lib/ActivityStreams.js

```diff
@@ -45,6 +46,7 @@ function ActivityStreams(options = {}) {
     this._tabTracker = new TabTracker(this.appURLs);
   }
   this._appURLHider = new AppURLHider(this.appURLs);
+  this._perfMeter = new PerfMeter(this.appURLs);
```

Comment:
only enable performance tracking if a performance profiling pref is turned on (or option passed)